scripts/ssl-manager.js

```diff
@@ -601,15 +646,17 @@ function SSLManager(config) {
         nodes = resp.nodes;
 
         for (var j = 0, node; node = nodes[j]; j++) {
+            if (!isBLNodeExists && nodeManager.isBalancerLayer(node.nodeGroup) && node.ismaster) {
+                nodeManager.setBalancerMasterNode(node);
+                isBLNodeExists = true;
+            }
             if (node.nodeGroup != group) continue;
             
             me.initAddOnExtIp(config.withExtIp);
 
             if (config.withExtIp) {
-                if (!node.extIPs || node.extIPs.length == 0) {
-                    resp = me.exec.call(nodeManager, nodeManager.attachExtIp, node.id);
-                    if (resp.result != 0) return resp;
-                }
+                targetNode = (isBLNodeExists) ? nodeManager.getBalancerMasterNode() : node;
```

Comment:
you can't check balancer node here

Reply:
removed. The same code executes in _getEntryPointGroup_ if it isn't a bl layer